### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

File: components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+.btn{
+    height:min-content;
+    width:min-content;
+    padding: 0.8rem;
+    border:none;
+    border-radius: 0.3rem;
+    color:#fefefe;
+    background-color: #fefefe;
```
Comment (OmKakatkar) [inline components/Toast/toast.css]: Try to use variables for colors

**Review by OmKakatkar — APPROVED** [review]

**Review by romabulani — APPROVED** [review]
good to merge

File: components/Toast/toast.css (hunk 1 above)
Comment (KamiyaGaikwad) [reply components/Toast/toast.css]: @OmKakatkar thank you for pointing it out, I will do the changes

**Comment by KamiyaGaikwad** [comment]
Thank you @romabulani @OmKakatkar for approving

**Pull request merged**